lib/versioning/VersionID.js

```diff
@@ -202,8 +215,12 @@ function base62Decode(str) {
         const enc2 = str.substring(start, start + B62V_EPAD.length);
         const orig2 = base62Integer.decode(enc2);
         start += B62V_EPAD.length;
-        const enc3 = str.substring(start);
+        let enc3 = str.substring(start);
+        // take off null bytes padding only from the end
+        const re = new RegExp('\u0000+$'); // eslint-disable-line
```

Comment:
as long as we are guaranteed to never have a null byte besides the padding at the end, we can do this. Benchmark is about 2x speedup in favor of slice

Reply:
> as long as we are guaranteed to never have a null byte besides the padding at the end, we can do this. Benchmark is about 2x speedup in favor of slice

why would we have a null byte ?

Reply:
oh I was thinking in case we add a 4th field in the future and use it a separator or something. But we would just use the offsets so just disregard that.